src/feature-test/user_sees_expanded_recipe_when_clicked.test.js

```diff
@@ -0,0 +1,53 @@
+import '../setupTest.js'
+import fetch from '../unit-test/mocks/recipeListFetch';
+
+describe('I.I.A.S', () => {
+  it('the user sees expanded recipe when clicking on title', async () => {
+    const browser = await puppeteer.launch({
+      // slowMo: 40,
+      // This two lines below allows the dev to see the flow on the chromium page
```

Comment:
I don't think we need to have these. They are useful when writing a test and debugging elements and watching a flow but once up on master I don't think we need it. They have served their purpose and if needed again can be added back.

Reply:
Good catch. They can definitely go